pkg/trace/pb/tracer_payload.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package pb;
+
+import "span.proto";
+
+// TraceChunk represents a list of spans with the same trace id.
+message TraceChunk {
+	int32 priority = 1; // Sampling priority. Example: AutoDrop(0), AutoKeep(1), UserKeep(2), etc.
+	string origin = 2; // Origin product. Example: "lambda", "rum", "ciapp-test", etc.
+	repeated Span spans = 3; // List of spans.
+	map<string, string> tags = 4; // Trace level tags - tags common in all `spans`.
+	bool droppedTrace = 5; // If true, the trace chunk was dropped, but some spans were kept by Event Processor.
+}
+
+// TracerPayload represents a payload the trace agent receives from tracers.
+message TracerPayload {
+	string containerID = 1; // ID of the container where the tracer is running on.
```

Comment:
```suggestion
	// containerID specifies the ID of the container where the tracer is running on.
	string containerID = 1; 
```
Please apply the following changes here:
* Rephrase the comments so that they start with the identifier name.
* Place them above the identifier (not to the right).

What we are after here is noticing the comments show up in the generated file too. You will notice that the comment for the top-level message `TracerPayload` and `TraceChunk` do show up, because you have gone by the logic in the above points.